test/MSVS/vs-mult-auto-guid.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
```

Comment:
Replace the entire header here with what's in template/test.py

Reply:
should be fixed